src/cmds/scripts/pbs_schema_upgrade

```diff
@@ -163,47 +163,59 @@ upgrade_pbs_schema_from_v1_3_0() {
 		
 		CREATE EXTENSION hstore;
 		
-		ALTER TABLE pbs.job ADD attributes public.hstore DEFAULT ''::public.hstore NOT NULL;
+		ALTER TABLE pbs.job ADD attributes public.hstore DEFAULT ''::public.hstore;
```

Comment:
As discussed if we implement my next comment our database layer is now capable of handling NULL or empty string. So no need to modify these sql queries a lot. We only need to make sure that we remove the NOT NULL constraint for the attributes column and have empty string as default.
Same comment applies for other sql queries in this file.

Reply:
This I can handle as part of the upcoming db layer change.